test/e2e/operation/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/migration/general/MySQLMigrationGeneralE2EIT.java

```diff
@@ -84,6 +84,9 @@ void assertMigrationSuccess(final PipelineTestParameter testParam) throws SQLExc
                     new E2EIncrementalTask(containerComposer.getSourceDataSource(), SOURCE_TABLE_NAME, new SnowflakeKeyGenerateAlgorithm(), containerComposer.getDatabaseType(), 30));
             TimeUnit.SECONDS.timedJoin(containerComposer.getIncreaseTaskThread(), 30);
             containerComposer.sourceExecuteWithLog(String.format("INSERT INTO %s (order_id, user_id, status) VALUES (10000, 1, 'OK')", SOURCE_TABLE_NAME));
+            // TODO If the fractional seconds > 0 , query from JDBC will thrown Zero date value prohibited SqlTimestampValueFactory.localCreateFromDatetime, zeroDateTimeBehavior not effect
+            DataSourceExecuteUtils.execute(containerComposer.getSourceDataSource(), String.format("UPDATE %s SET t_datetime='0000-00-00 00:00:00.000000', t_timestamp='0000-00-00 00:00:00', "
+                    + "t_time='00:00:00.0' WHERE order_id = 1", SOURCE_TABLE_NAME));
```

Comment:
If `0000-00-00 00:00:00.123` could not be read by MySQL JDBC driver, then it should be corrected firstly, it's not necessary to be compatible with it.